charts/rhtap-infrastructure/templates/openshift-pipelines/tekton-chains-password.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if .Values.infrastructure.openShiftPipelines.enabled }}
```

Comment:
This resource feels superfluous.

Reply:
That's the link with [RHDH toggle](https://github.com/otaviof/rhtap-cli/blob/9fb0557a67891d058ce7c9342db9eb58b1ac5b0b/charts/values.yaml.tpl#L107-L108). So, when RHDH is disabled we can also disable OpenShift Pipelines changes.